test/e2e/toolbox_test.go

```diff
@@ -257,9 +257,9 @@ var _ = Describe("Toolbox-specific testing", func() {
 		var username string = "testuser"
 		var homeDir string = "/home/testuser"
 		var shell string = "/bin/bash"
-		var uid string = "2000"
+		var uid string = "1411"
```

Comment:
Relevant Cirrus code: https://github.com/containers/podman/blob/93447e292dc3166b2f699ba1ce75666448ccfab0/contrib/cirrus/lib.sh#L162-L165

I briefly considered adding a comment here, "make sure this is <1500", with an explanation... but it really doesn't seem worth the clutter. Much as I love warnings in comments, this is not code that's likely to ever be touched again.